### PR TITLE
added date of birth field while signing up

### PR DESCRIPTION
**Pull request by Tauqeer-Ahmed-99**
added date of birth field while signing up

### CODE REVIEW

File: src/screens/LoginScreen/LoginScreen.tsx [hunk 1]
```diff
@@ -10,6 +10,8 @@ const LoginScreen = () => {
 
   const navigate = useNavigate();
 
+  const birtDateRef = useRef() as React.MutableRefObject<HTMLInputElement>;
```
Comment (Tanvir04khan) [inline src/screens/LoginScreen/LoginScreen.tsx]: BirthDateRef

**Review by Tanvir04khan — DISMISSED** [review]
LGTM

File: src/screens/LoginScreen/LoginScreen.tsx (hunk 1 above)
Comment (Tauqeer-Ahmed-99) [reply src/screens/LoginScreen/LoginScreen.tsx]: Changing

Comment (Tauqeer-Ahmed-99) [reply src/screens/LoginScreen/LoginScreen.tsx]: Changed @Tanvir04khan

**Review by Tanvir04khan — APPROVED** [review]
LGTM

**Pull request merged**